### ISSUE TITLE
jqwik should fail property test instead of ignoring it

### ISSUE THREAD

**Issue by michaelmosmann** [jlink/jqwik#54]
## Testing Problem

following kotlin test 

```kotlin
class FooTest {

  @Property
  fun foo(name: String, @ForAll other: String) {
    assertNull(name)
  }
}
```

should fail, but is ignored (guess: name is not annotated but other is).

## Suggested Solution

jqwik should fail such a test.

**Comment by jlink** [comment]
Well, the test will be marked as ignored together with an ignore message "All parameters must have @ForAll annotation". Depending on your IDE / build tool configuration this can be as obvious as a test failure. What additional information would a test failure provide?

**Comment by michaelmosmann** [comment]
For me it would be ok to fail this test with this message. AFAIK gradle reports an skipped test (if reporting) is enabled. But if i want gradle to fail the build i must distinguish between this case and a disabled junit test.

For me this is some kind of configuration error.. like @ForAll for a type without an known Arbitrary. It should fail, because i would not expect, that the test will be skipped.

**Comment by michaelmosmann** [comment]
Thanks a lot:)

**Comment by jlink** [comment]
Fixed in https://github.com/jlink/jqwik/commit/d1f899cb18ba2757190443152172397bf74e59f1

**Comment by jlink** [comment]
Available in 1.1.3-SNAPSHOT

**Comment by jlink** [comment]
Released in 1.1.3
